docs/sphinx/specification/quake-dialect.md

```diff
@@ -148,9 +148,3 @@ Mmeory                          Value
 We know that one Hadamard is applied to `%q0_L` and generates `%q0_M`, and
 the other is applied `%q0_Y` and generates `%q0_Z`.  Hence, there is no
 connection between them---which means they cannot cancel each other out.
-
-## Quake Types
```

Comment:
Why remove? It is part of the dialect.

Reply:
The text is not clear enough to be useful in my opinion. It used to be longer, but was outdated. The two sentences are all that is left of it and one of them is likely to cause more confusion than clarify what the type is (specifically the one stating a wire is an "intermediate state of a single qubit in time" - as per the discussions we had, wires have nothing to do with state).

Reply:
OK. Actual disinformation isn't helpful, I totally agree.

On the other hand, documenting the dialect's type system seems like a good idea as leaving out any description doesn't seem helpful.

Maybe we need an issue to track this or a link to other documentation? Not sure.

Reply:
Seems fair to track it in an issue. I am not overly eager to publish more docs due to how rapidly we have been iterating on the dialects, but if we set the right expectations regarding stability, it would be nice to include more docs: https://github.com/NVIDIA/cuda-quantum/issues/727